base_rest/tests/common.py

```diff
@@ -24,6 +26,13 @@
 from ..core import RestServicesRegistry, _rest_services_databases
 from ..tools import _inspect_methods
 
+if sys.version_info < (3, 8):
+    from odoo.tests.common import TreeCase
```

Comment:
@fdegrave Can you also keep the comment
```
   # This commit
    # https://github.com/odoo/odoo/commit/ed831abd7d91ee873a1a92ecea2de316a025754f
    # introduced a backward imcompatible change
    # that makes simple unit tests ran through Odoo suite fail
    # because `doClassCleanups` is not defined.
    # They defined it on `TreeCase` so... there we go.
```